src/Crypt32.Shared/Crypt32+CRYPT_PROV_TYPE.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace PInvoke
```

Comment:
Please confirm that you don't get any build warnings. In this case, I expect a stylecop warning because you're missing the copyright notice at the top of the file. The squigglies in VS should help you automatically correct these with little effort on your part.